Mage.Sets/src/mage/cards/b/BeneathTheSands.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
```

Comment:
Also, you'll need perl to run it - possibly strawberry perl if you're on windoze